cmd/actions.go

```diff
@@ -36,6 +37,27 @@ var (
 			},
 		},
 	}
+
+	subcmdActionsSetRunnerToken = &cli.Command{
+		Name:    "set-runner-token",
+		Usage:   "Set a new token for a runner to as register token",
+		Action:  runSetActionsRunnerToken,
+		Aliases: []string{"srt"},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "scope",
+				Aliases: []string{"s"},
+				Value:   "",
+				Usage:   "{owner}[/{repo}] - leave empty for a global runner",
+			},
+			&cli.StringFlag{
+				Name:    "token",
+				Aliases: []string{"t"},
+				Value:   "",
+				Usage:   "[{token}] - leave empty will generate a new token, otherwise will update the token to database. The token MUST be a 40 digital string containing only [0-9a-zA-Z]",
```

Comment:
Without a tool, it is difficult for end users to generate it.

The whole design should be like this:

```
./gitea actions generate-runner-token --global # generate and update
./gitea actions generate-runner-token --scope owner/repo  # generate and update
./gitea actions generate-runner-token --display-only # display a token only
./gitea actions generate-runner-token --global --set-token the-token-value # set the token to the pre-generated one
./gitea actions generate-runner-token --scope owner/repo --set-token the-token-value # set the token to the pre-generated one
```

Do not introduce more technical debits

Reply:
This is my previous solution. I created a new subcommand now because you said `generate-runner-token` is conflicted with `set-token`(previous put-token).

Reply:
> This is my previous solution. I created a new subcommand now because you said `generate-runner-token` is conflicted with `set-token`(previous put-token).

That's not your previous solution. At least I can see huge difference.

Reply:
The differences:

1. we should forbid the `./gitea actions generate-runner-token` without arguments (maybe it is a breaking change, but I believe we should do it, and it is easy for end users to follow)
2. we should add `--display-only` to help end users to generate the token
3. `--set-token` only works with `--global` or `--scope`